modules/bitgo/src/v2/coins/near.ts

```diff
@@ -45,28 +45,47 @@ const nearUtils = accountLib.Near.Utils.default;
 const HEX_REGEX = /^[0-9a-fA-F]+$/;
 
 export class Near extends BaseCoin {
-  constructor(bitgo: BitGo) {
+  protected readonly _staticsCoin: Readonly<StaticsBaseCoin>;
+  constructor(bitgo: BitGo, staticsCoin?: Readonly<StaticsBaseCoin>) {
     super(bitgo);
+
+    if (!staticsCoin) {
+      throw new Error('missing required constructor parameter staticsCoin');
+    }
+
+    this._staticsCoin = staticsCoin;
+  }
+
+  static createInstance(bitgo: BitGo, staticsCoin?: Readonly<StaticsBaseCoin>): BaseCoin {
+    return new Near(bitgo, staticsCoin);
   }
 
-  static createInstance(bitgo: BitGo): BaseCoin {
-    return new Near(bitgo);
+  allowsAccountConsolidations(): boolean {
+    return true;
   }
 
+    /**
+   * Flag indicating if this coin supports TSS wallets.
+   * @returns {boolean} True if TSS Wallets can be created for this coin
+   */
+     supportsTss(): boolean {
+      return true;
+    }
+
   getChain(): string {
-    return 'near';
+    return this._staticsCoin.name;
   }
 
   getBaseChain(): string {
-    return 'near';
+    return this.getChain();
   }
 
-  getFamily(): string {
-    return 'near';
+  getFamily(): CoinFamily {
+    return this._staticsCoin.family;
   }
 
   getFullName(): string {
-    return 'Near';
+    return this._staticsCoin.fullName;
   }
 
   getBaseFactor(): any {
```

Comment:
Modify this method too

`return Math.pow(10, this._staticsCoin.decimalPlaces)`